src/+datools/+examples/+sandu/l63_experiments.m

```diff
@@ -62,24 +63,23 @@
 modelODE.TimeSpan = [0, Delta_t];
 
 % Propogate the model
-[tt, yy] = ode45(natureODE.Rhs.F, [0 10], nature0);
+[tt, yy] = ode45(natureODE.Rhs.F, [0, 10], nature0);
 natureODE.Y0 = yy(end, :).';
 
 % initialize model
-model  = datools.Model('Solver', solvermodel, 'ODEModel', modelODE);
+model = datools.Model('Solver', solvermodel, 'ODEModel', modelODE);
 nature = datools.Model('Solver', solvernature, 'ODEModel', natureODE);
 
 % Observation Model
-naturetomodel = datools.observation.Linear(numel(nature0), 'H',...
+naturetomodel = datools.observation.Linear(numel(nature0), 'H', ...
     speye(natureODE.NumVars));
 
 % observe these variables
 % change the array if you want to observe lesser state variables
 observeindicies = 1:1:natureODE.NumVars;
-
 nobsvars = numel(observeindicies);
 
-R = variance*(1/1)*speye(nobsvars);
+R = variance * (1 / 1) * speye(nobsvars);
```

Comment:
why is there a 1/1???

Reply:
I have no idea. it was there in the original script from where i copied. I will remove it.